test/dynamo/test_exc.py

```diff
@@ -166,7 +166,7 @@ def fn001(x):
         self.assertExpectedInline(
             munge_exc(record.getMessage()),
             """\
-Graph break: 'skip function graph_break in file _dynamo/decorators.py'', skipped according skipfiles.SKIP_DIRS' from user code at:
```

Comment:
This is because ```trace_rules.lookup``` returns class ```SkipFilesVariable```, which doesn't include the particular reason of the skipping. We are going to merge all in_graph/skip/inline rule check into this single point, so my plan is to log the reason of in_graph/skip/inline during looking up instead of passing it to the variable constructor. I'll do the logging in the unifying API PR.